server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java

```diff
@@ -724,32 +724,6 @@ public long holdTime() {
     }
   }
 
-  public void close() {
```

Comment:
Looking around the code, it seems one place it would make sense to call this function is in the `TabletServer.run()` function.  However I would not want to add the call there without a good understanding of how it may impact shutdown scenarios.

Reply:
Looks like it hasn't been called by TabletServer in our git history.  At this point it almost feels safer to remove it since we are not sure of the consequences of calling it.

Reply:
And to answer your question, no it is not overriding an AutoCloaseable close method.